docs/reference/commandline/create.md

```diff
@@ -68,7 +68,11 @@ Creates a new container.
       -u, --user=""                 Username or UID
       --ulimit=[]                   Ulimit options
       --uts=""                      UTS namespace to use
-      -v, --volume=[]               Bind mount a volume
+      -v, --volume=[]               Bind mount a volume with: [host-dir:]container-dir[:<options>], where
```

Comment:
@HackToday you're right in that we don't mention that in other places. Yes, the volume-name is just a plain-string, a basic description can be found here; https://github.com/docker/docker/blob/master/docs/reference/run.md#volume-shared-filesystems